.packit.yaml

```diff
@@ -9,7 +9,7 @@ packages:
     actions:
       create-archive:
         - spectool -g -s0 rust-atuin.spec
-        - sh -c 'rust2rpm --vendor --stdout atuin > /dev/null'
+        - sh -c 'rust2rpm --vendor atuin > /dev/null'
```

Comment:
Can remove the `> /dev/null` in this case as well.

Also note that we need to switch from `crate` source to git repo because the repo contains multiple crate projects.

Reply:
By "switch from `crate` source to git repo" do you mean that the action should clone down the repo and then invoke rust2rpm against the cloned directory?

Reply:
No, I mean that the `Source0` in the spec file will point to the git archive. This is when we change away from `vendor` and move to handling all of the dependencies, so at that point the `rust2rpm` step can also be removed.